CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 2.51.1 (2021-11-02)
```

Comment:
This should be `2.52.0` since it's more akin to a feature than a bug fix. You're also missing the updates to `package.json` and `package-lock.json` to set the new version number.

Reply:
You told me that last time too, sorry. Fixed now!